pkg/server/server.go

```diff
@@ -457,3 +484,32 @@ func (h *handler) renderTemplate(template string, identity *token.Identity) (str
 
 	return template, nil
 }
+
+func (h *handler) CallBackForFileLoad(dynamicContent []byte) error {
+	var backendModes BackendModeConfig
+	logrus.Infof("BackendMode dynamic file got changed to %s", string(dynamicContent))
+	err := json.Unmarshal(dynamicContent, &backendModes)
+	if err != nil {
+		logrus.Infof("CallBackForFileLoad: could not unmarshal dynamic file.")
+		return err
+	}
+	if h.backendMapper.currentModes != backendModes.BackendMode {
```

Comment:
this seems too sensitive to typos/whitespace in the config, what if it's "MountedFile DynamicFile" vs "MountedFile  DynamicFile"? since the file is json, what about json array? Or comma-separated string to be consistent with cfg.BackendMode

Reply:
thank you for the comments! if there is typo in the dynamic backend json file, user can modify it quickly? I will add it in the release note for the format. does it sound good to you?